locale/en/docs/guides/nodejs-docker-webapp.md

```diff
@@ -141,7 +144,10 @@ FROM node:boron
 WORKDIR /usr/src/app
 
 # Install app dependencies
-COPY package.json package-lock.json .
+COPY package.json .
+# For npm@5 or later, copy package-lock.json as well
```

Comment:
Can we use the same style for the comment? One blank line before, no blank line below. Or change the previous one to be consistent with this one :)